--- a/CMakeLists.txt	2022-05-01 07:05:20.000000000 +0200
+++ b/CMakeLists.txt	2022-09-24 11:48:07.658454874 +0200
@@ -440,7 +440,7 @@
 endif ()
 
 
-add_library(freetype
+add_library(Freetype
   ${PUBLIC_HEADERS}
   ${PUBLIC_CONFIG_HEADERS}
   ${PRIVATE_HEADERS}
@@ -448,30 +448,30 @@
 )
 
 set_target_properties(
-  freetype PROPERTIES
+  Freetype PROPERTIES
     C_VISIBILITY_PRESET hidden)
 
 target_compile_definitions(
-  freetype PRIVATE FT2_BUILD_LIBRARY)
+  Freetype PRIVATE FT2_BUILD_LIBRARY)
 
 if (WIN32)
   target_compile_definitions(
-    freetype PRIVATE _CRT_SECURE_NO_WARNINGS _CRT_NONSTDC_NO_WARNINGS)
+    Freetype PRIVATE _CRT_SECURE_NO_WARNINGS _CRT_NONSTDC_NO_WARNINGS)
   if (BUILD_SHARED_LIBS)
     target_compile_definitions(
-      freetype PRIVATE DLL_EXPORT)
+      Freetype PRIVATE DLL_EXPORT)
   endif ()
 endif ()
 
 if (BUILD_SHARED_LIBS)
-  set_target_properties(freetype PROPERTIES
+  set_target_properties(Freetype PROPERTIES
     VERSION ${LIBRARY_VERSION}
     SOVERSION ${LIBRARY_SOVERSION})
 endif ()
 
 # Pick up ftconfig.h and ftoption.h generated above, first.
 target_include_directories(
-  freetype
+  Freetype
     PUBLIC
       $<INSTALL_INTERFACE:include/freetype2>
       $<BUILD_INTERFACE:${CMAKE_CURRENT_BINARY_DIR}/include>
@@ -489,7 +489,7 @@
   set_property(SOURCE ${PUBLIC_CONFIG_HEADERS}
     PROPERTY MACOSX_PACKAGE_LOCATION Headers/config
   )
-  set_target_properties(freetype PROPERTIES
+  set_target_properties(Freetype PROPERTIES
     FRAMEWORK TRUE
     MACOSX_FRAMEWORK_INFO_PLIST ${CMAKE_CURRENT_SOURCE_DIR}/builds/mac/freetype-Info.plist
     PUBLIC_HEADER "${PUBLIC_HEADERS}"
@@ -504,13 +504,13 @@
 set(PKGCONFIG_LIBS_PRIVATE "")
 
 if (ZLIB_FOUND)
-  target_link_libraries(freetype PRIVATE ${ZLIB_LIBRARIES})
-  target_include_directories(freetype PRIVATE ${ZLIB_INCLUDE_DIRS})
+  target_link_libraries(Freetype PRIVATE ${ZLIB_LIBRARIES})
+  target_include_directories(Freetype PRIVATE ${ZLIB_INCLUDE_DIRS})
   list(APPEND PKGCONFIG_REQUIRES_PRIVATE "zlib")
 endif ()
 if (BZIP2_FOUND)
-  target_link_libraries(freetype PRIVATE ${BZIP2_LIBRARIES})
-  target_include_directories(freetype PRIVATE ${BZIP2_INCLUDE_DIR}) # not BZIP2_INCLUDE_DIRS
+  target_link_libraries(Freetype PRIVATE ${BZIP2_LIBRARIES})
+  target_include_directories(Freetype PRIVATE ${BZIP2_INCLUDE_DIR}) # not BZIP2_INCLUDE_DIRS
   if (PC_BZIP2_FOUND)
     list(APPEND PKGCONFIG_REQUIRES_PRIVATE "bzip2")
   else ()
@@ -518,20 +518,20 @@
   endif ()
 endif ()
 if (PNG_FOUND)
-  target_link_libraries(freetype PRIVATE ${PNG_LIBRARIES})
-  target_compile_definitions(freetype PRIVATE ${PNG_DEFINITIONS})
-  target_include_directories(freetype PRIVATE ${PNG_INCLUDE_DIRS})
+  target_link_libraries(Freetype PRIVATE ${PNG_LIBRARIES})
+  target_compile_definitions(Freetype PRIVATE ${PNG_DEFINITIONS})
+  target_include_directories(Freetype PRIVATE ${PNG_INCLUDE_DIRS})
   list(APPEND PKGCONFIG_REQUIRES_PRIVATE "libpng")
 endif ()
 if (HarfBuzz_FOUND)
-  target_link_libraries(freetype PRIVATE ${HarfBuzz_LIBRARY})
-  target_include_directories(freetype PRIVATE ${HarfBuzz_INCLUDE_DIRS})
+  target_link_libraries(Freetype PRIVATE ${HarfBuzz_LIBRARY})
+  target_include_directories(Freetype PRIVATE ${HarfBuzz_INCLUDE_DIRS})
   list(APPEND PKGCONFIG_REQUIRES_PRIVATE "harfbuzz >= ${HARFBUZZ_MIN_VERSION}")
 endif ()
 if (BROTLIDEC_FOUND)
-  target_link_libraries(freetype PRIVATE ${BROTLIDEC_LIBRARIES})
-  target_compile_definitions(freetype PRIVATE ${BROTLIDEC_DEFINITIONS})
-  target_include_directories(freetype PRIVATE ${BROTLIDEC_INCLUDE_DIRS})
+  target_link_libraries(Freetype PRIVATE ${BROTLIDEC_LIBRARIES})
+  target_compile_definitions(Freetype PRIVATE ${BROTLIDEC_DEFINITIONS})
+  target_include_directories(Freetype PRIVATE ${BROTLIDEC_INCLUDE_DIRS})
   list(APPEND PKGCONFIG_REQUIRES_PRIVATE "libbrotlidec")
 endif ()
 
@@ -614,7 +614,7 @@
     COMPATIBILITY SameMajorVersion)
 
   install(
-    TARGETS freetype
+    TARGETS Freetype
       EXPORT freetype-targets
       LIBRARY DESTINATION ${CMAKE_INSTALL_LIBDIR}
       ARCHIVE DESTINATION ${CMAKE_INSTALL_LIBDIR}
@@ -623,6 +623,7 @@
       COMPONENT libraries)
   install(
     EXPORT freetype-targets
+      NAMESPACE Freetype::
       DESTINATION ${CMAKE_INSTALL_LIBDIR}/cmake/freetype
       FILE freetype-config.cmake
       COMPONENT headers)
