diff --git a/build/cmake/CMakeLists.txt b/build/cmake/CMakeLists.txt
index 419c243..4ddb08f 100644
--- a/build/cmake/CMakeLists.txt
+++ b/build/cmake/CMakeLists.txt
@@ -36,7 +37,11 @@ set (CMAKE_ARCHIVE_OUTPUT_DIRECTORY ${CMAKE_BINARY_DIR}/lib)
   set (CMAKE_RUNTIME_OUTPUT_DIRECTORY ${CMAKE_BINARY_DIR}/bin)
 endif()
 
-find_package (OpenGL REQUIRED)
+if (APPLE)
+  set (OPENGL_LIBRARIES "-framework OpenGL")
+else ()
+  find_package (OpenGL REQUIRED)
+endif ()
 
 # prefer GLVND
 if (OPENGL_opengl_LIBRARY)
