--- a/ports/cmake/src/CMakeLists.txt	2026-05-02 18:48:58.499606030 +0200
+++ b/ports/cmake/src/CMakeLists.txt	2026-05-02 18:49:08.897226176 +0200
@@ -196,7 +196,7 @@
 check_cpu_arch_arm32(ARCH_IS_ARM32)
 check_cpu_arch_arm64(ARCH_IS_ARM64)
 
-if(WIN32 OR (ARCH_IS_ARM64 AND APPLE))
+if(WIN32 OR ARCH_IS_ARM64)
     set(HAVE_FPU 1)
 else()
     cmake_host_system_information(RESULT HAVE_FPU QUERY HAS_FPU)
